cumulus/tasks/queue-pdrs/package.json

```diff
@@ -38,6 +38,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@cumulus/common": "^1.0.1",
+    "@cumulus/cumulus-message-adapter-js": "0.0.1-beta.3",
```

Comment:
We should probably have this on 1.0.0 now.

Reply:
https://bugs.earthdata.nasa.gov/browse/CUMULUS-367